packages/docs/docs/sdk/Kotlin/android/io.logto.sdk.android/-logto-client/sign-in.md

```diff
@@ -0,0 +1,15 @@
+# signIn
+
+
+fun signIn(context: [Activity](https://developer.android.com/reference/kotlin/android/app/Activity.html), redirectUri: [String](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-string/index.html), completion: [EmptyCompletion](../../io.logto.sdk.android.completion/-empty-completion/index.md)&lt;[LogtoException](../../io.logto.sdk.android.exception/-logto-exception/index.md)&gt;)
+
+Sign in
+
+## Parameters
+
+
+| | |
```

Comment:
nit: this can be replaced with `| Name | Description |` globally?

Reply:
replaced with `| Name | Summary |` to align with the defaults.